sources/packages/api/src/services/education-program-offering/education-program-offering-validation.service.ts

```diff
@@ -0,0 +1,131 @@
+import { Injectable } from "@nestjs/common";
+import { OfferingStatus } from "../../database/entities";
+import {
+  OfferingValidationResult,
+  SaveOfferingModel,
+  ValidationWarning,
+  ValidationWarningResult,
+} from "./education-program-offering-validation.models";
+import { validateSync, ValidationError } from "class-validator";
+import { plainToClass } from "class-transformer";
+import { flattenErrors } from "../../utilities/class-validation";
+import { CustomNamedError } from "../../utilities";
+import { OFFERING_VALIDATION_CRITICAL_ERROR } from "../../constants";
+
+/**
+ * Executes the validations on an offering model.
+ */
+@Injectable()
+export class EducationProgramOfferingValidationService {
+  validateOfferingModel(
```

Comment:
Comment to this method? I know it will be the same as validateOfferingModels.

Reply:
I forgot this one 👍

Reply:
Comment added.